lib/elixir/lib/exception.ex

```diff
@@ -1639,6 +1632,22 @@ defmodule ErlangError do
     %KeyError{key: key, term: map}
   end
 
+  def normalize({:baddot, true, key, map}, _stacktrace) do
+    %ArgumentError{
+      message:
+        "could not find key #{inspect(key)} on #{inspect(map)}. " <>
+          "When using the map.field syntax, make sure the left side of the dot is a map"
```

Comment:
Not a native speaker here, but I've mostly seen this referred to as

```suggestion
          "When using the map.field syntax, make sure the left-hand side of the dot is a map"
```

?

Reply:
Wikipedia [agrees](https://en.wikipedia.org/wiki/Sides_of_an_equation), sounds good.

I copied this from the existing hint, after grepping through the codebase it seems we are currently using all of the following:
- left side
- left hand side
- left-hand side (the correct one)

We might want to fix the existing ones as well in a separate PR, WDYT?